Doc/c-api/init.rst

```diff
@@ -221,6 +221,13 @@ to 1 and ``-bb`` sets :c:data:`Py_BytesWarningFlag` to 2.
    Set by the :option:`-v` option and the :envvar:`PYTHONVERBOSE` environment
    variable.
 
+.. c:var:: int Py_UTF8Mode
+
+   Enables the UTF-8 Mode.
```

Comment:
You can add a link to https://docs.python.org/dev/library/os.html#python-utf-8-mode using ``utf8-mode`` reference. Please replace "UTF-8 Mode" with "Pyhon UTF-8 Mode".